linetools/spectralline.py

```diff
@@ -543,7 +543,23 @@ def measure_aodm(self, nsig=3., normalize=True):
         # Log
         laa.log_clm(self.attrib)
 
-    # Output
+    def __getitem__(self, k):
+        """  Passback a data bit from the Line
+        """
+        try:
+            return getattr(self,k)
+        except AttributeError:
+            try:
+                return self.attrib[k]
+            except KeyError:
+                try:
+                    return self.analy[k]
+                except KeyError:
+                    try:
+                        return self.data[k]
+                    except KeyError:
+                        return None
+
```

Comment:
I worry that this much flexibility will produce bugs that are difficult to track down. Do you need to be able to access everything here (analy, attrib, & data)? Or would just having one or two deal with most use cases?